examples/src/models/chat/integration_openai_stream_tokens.ts

```diff
@@ -0,0 +1,30 @@
+import { AIMessageChunk } from "@langchain/core/messages";
+import { ChatOpenAI } from "@langchain/openai";
+
+// Instantiate the model
+const model = new ChatOpenAI();
+
+const response = await model.stream("Hello, how are you?", {
+  // Pass the stream options
+  stream_options: {
```

Comment:
We've been using camel case for this kind of thing to match JS conventions (also `includeUsage` below)